nodejs/week1/circle.js

```diff
@@ -0,0 +1,21 @@
+class circle {
+          constructor(radius) {
+                    this.radius = radius;
+          }
+
+          getDiameter() {
+                    return 2 * this.radius;
+          }
+          getArea() {
+                    return Math.PI * this.radius * this.radius;
```

Comment:
It is also a good idea to avoid repetitive code `this.radius * this.radius` . You can use  `Math.pow(this.radius, 2)` instead; 
You can read about Math. pow() here : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/pow